### DISCUSSION TITLE
How was the javadoc demo made?

### DISCUSSION THREAD

**Discussion by ralismark** [L3MON4D3/LuaSnip#452] [Q&A]
I'm really interested in the method snippet used in the [demo video on the readme](https://github.com/L3MON4D3/LuaSnip#luasnip). Is there code for that snippet?

**Comment by L3MON4D3** [comment]
The original code is in the [doc](https://github.com/L3MON4D3/LuaSnip/blob/master/DOC.md#dynamicnode), but you can find a better version [here](https://github.com/L3MON4D3/LuaSnip/pull/228#issuecomment-981505317) :)
